### PR TITLE
Feature/#1 waiter view

### PR DESCRIPTION
**Pull request by Nadiajayy**
Couldn't get Discount working in my view on Order Panel hence commented

### CODE REVIEW

File: routes/waiter/waiter.controller.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const Menu = require('./waiter.model');
```
Comment (last1here) [inline routes/waiter/waiter.controller.js:1]: Not sure the `/waiter/*` files are needed? seem to be a clone of what is implemented in the `/menu/*` files?

File: client/app/waiter/waiter.component.html [hunk 2]
```diff
@@ -1,23 +1,148 @@
 <div class="container-fluid">
-    <!-- https://getbootstrap.com/docs/4.3/layout/overview/ and 
+  <!-- https://getbootstrap.com/docs/4.3/layout/overview/ and
```
Comment (dorusomcutean) [inline client/app/waiter/waiter.component.html:2]: These are not needed anymore

File: client/app/waiter/waiter.component.html [hunk 3]
```diff
@@ -1,23 +1,148 @@
...
+      <button class="openbtn" (click)="openMenuNav()">Tables</button>
+    </div>
+
+    <div class="col content pr-0" id="summary">
+      <div class="row">
+        <div *ngFor="let staff of staff">
+          <h3>Waiter View- {{staff.firstname}}</h3>
+          <hr>
         </div>
+
```
Comment (dorusomcutean) [inline client/app/waiter/waiter.component.html:43]: no need for so much empty spae here

File: client/app/order-panel/order-panel.component.ts [hunk 4]
```diff
@@ -71,11 +71,11 @@ export class OrderPanelComponent implements OnInit {
      * Calculate the total of all prices. Item price * quantity
      */
     calculateTotal() {
-        this.totalNoDiscount = this.order.items.reduce(
-            (accumulator, element) =>
-                element.price + accumulator,
-            0
-        );
+        //this.totalNoDiscount = this.order.items.reduce(
```
Comment (last1here) [inline client/app/order-panel/order-panel.component.ts]: Uncomment.

File: client/app/waiter/waiter.component.css [hunk 5]
```diff
@@ -0,0 +1,249 @@
+@import "~bootstrap/dist/css/bootstrap.css";
```
Comment (last1here) [inline client/app/waiter/waiter.component.css]: Not needed see #55.

File: client/app/waiter/waiter.component.css [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+ /* Style the tab content (and add height:100% for full page content) */
+ .tabcontent {
+   color: white;
+   display: none;
+   padding: 50px 20px;
+   height: 290%;
+   width: 100%;
+ }
+
+ #Starter {background-color: grey;}
```
Comment (last1here) [inline client/app/waiter/waiter.component.css:208]: nit: use a generic class.

- can be done later.

File: client/app/waiter/waiter.component.html [hunk 7]
```diff
@@ -1,23 +1,158 @@
...
+      <div class="row">
+        <div *ngFor="let staff of staff">
+          <h3>Waiter View- {{staff.firstname}}</h3>
+        </div>
+      </div>
+      <div class="row">
+        <div id="main" class="tables">
+          <div *ngFor="let table of tables">
+            <hr>
+            <button [ngClass]=“{‘btn-danger’: table.order != null}” type="button" class="btn btn-secondary btn-lg btn-block" (click)="startOrderClick(table)">Table {{table.id}}</button>
```
Comment (last1here) [inline client/app/waiter/waiter.component.html]: Strange quotes.

File: client/app/waiter/waiter.component.html [hunk 8]
```diff
@@ -1,23 +1,158 @@
...
+          </div>
         </div>
         <div class="col-sm-12 col-md-4 col-lg-4" *ngIf="!!sidebarOrder">
-            <app-order-panel [waiterId]="waiterId" [setOrder]="sidebarOrder" (close)="sidebarOrder = null">
-            </app-order-panel>
+          <app-order-panel [waiterId]="waiterId" [type]="counterType" [setOrder]="sidebarOrder" (close)="sidebarOrder = null">
```
Comment (last1here) [inline client/app/waiter/waiter.component.html]: `counterType` is not defined in your component ts.

File: client/app/waiter/waiter.component.ts [hunk 9]
```diff
@@ -21,30 +24,61 @@ export class WaiterComponent implements OnInit, OnDestroy {
...
+
+    public staff = [
+        {
+            firstname: 'Klaus',
+            surname: 'Logan',
+            role: 'Waiter'
+        }
     ];
 
+    public menu = [
```
Comment (last1here) [inline client/app/waiter/waiter.component.ts]: No longer used.

File: client/app/waiter/waiter.component.ts [hunk 10]
```diff
@@ -56,57 +90,103 @@ export class WaiterComponent implements OnInit, OnDestroy {
...
     }
 
-    // Method needs to exist for untilDestroyed to work as expected in prod
-    // builds.
-    ngOnDestroy() { }
+
+
+
+
+/**
```
Comment (last1here) [inline client/app/waiter/waiter.component.ts]: formatting: Tab this next section in.

**Comment by last1here** [comment]
LGTM 🚀

**Pull request merged**